README.md

```diff
@@ -151,204 +167,10 @@ chainweb-nodes and collecting the logs from all nodes:
 mkdir -p tmp/run-nodes-logs
 
 # the first argument is the path to the chainweb-node binary
-./scripts/run-nodes.sh ./chainweb-node 20 ./tmp/run-nodes-logs
+./scripts/run-nodes.sh ./chainweb-node 10 ./tmp/run-nodes-logs
```

Comment:
Why is `run-nodes` here? The heading is "Configuring a chainweb-node"

Reply:
You're right, `run-nodes.sh` should be mentioned in "Running Chainweb".